Pod/Classes/OAStackView.m

```diff
@@ -141,6 +141,16 @@ - (void)setAlignmentConstraints {
   [self.alignmentStrategy alignLastView:self.subviews.lastObject];
 }
 
+- (void)setAlignmentStrategy:(OAStackViewAlignmentStrategy *)alignmentStrategy {
+  [_alignmentStrategy removeAddedConstraints];
```

Comment:
Hi @mattiasjahnke sorry for lat reply!

Since it is setter, do you think it worth it to check previous value of _alignmentStrategy before setting new one to keep semantics of setter unchanged?